src/StatsGatherer.js

```diff
@@ -137,7 +137,17 @@ class StatsGatherer extends EventEmitter {
 
       // TODO: for 2.0 - remove `lost` which is an integer of packets lost,
       // and use only `loss` which is percentage loss
-      const trackInfo = { track, kind, bitrate, lost, muted, loss, intervalLoss };
+      const trackInfo = {
+        track,
+        kind,
+        bitrate,
+        lost,
+        muted,
+        loss,
+        intervalLoss,
+        bytesSent: parseInt(report.bytesSent, 10),
```

Comment:
I think... make these consts above, and do `parseInt(report.bytesSent, 10) || -1`